client/src/components/Page/ChooseYourRunPage.js

```diff
@@ -4,7 +4,7 @@ import AppHeader from "../../components/AppHeader/AppHeader";
 import RunCard from "../../components/RunCard/RunCard";
 import BottomNav from "../../components/BottomNav/BottomNav";
 import Search from "../../components/Search/Search";
-import { getRuns, searchForKm } from "../../api/runs";
+import { deleteRunsById, getRuns, searchForKm } from "../../api/runs";
```

Comment:
```suggestion
import { getRuns, searchForKm } from "../../api/runs";
```

you can remove the function from the imports here

Reply:
Thanks